scripts/testpack.sh

```diff
@@ -4,7 +4,8 @@
 ROOT="$(pwd)/$(dirname "$0")/.."
 cd "$ROOT" || exit 1
 
-TEST_DIR=/tmp/test-pack
+TEST_DIR=$(mktemp -d)
```

Comment:
Set up the trap right after this line so it takes effect if the script exits somewhere in the middle